tools/mixmodel4repeated_measures/mixmodel.xml

```diff
@@ -0,0 +1,215 @@
+<tool id="mixmodel" name="mixmodel" version="3.0.0+galaxy0">
+    <description>ANOVA for repeated measures statistics</description>
+
+    <requirements>
+        <requirement type="package" version="1.1_5">r-batch</requirement>
+        <requirement type="package" version="1.1_13">r-lme4</requirement>
+        <requirement type="package" version="2.0_33">r-lmertest</requirement>
+        <requirement type="package" version="1.2_18">r-Matrix</requirement>
+        <requirement type="package" version="7.3_51.4">r-MASS</requirement>
+        <requirement type="package" version="3.2_0">r-ggplot2</requirement>
+        <requirement type="package" version="3.6_2">r-grid</requirement>
+        <requirement type="package" version="2.3">r-gridExtra</requirement>
+        <requirement type="package" version="2.40.0">bioconductor-multtest</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" level="fatal" />
+    </stdio>
+
+    <command><![CDATA[
+        Rscript $__tool_directory__/mixmodel_wrapper.R
+
+        dataMatrix_in "$dataMatrix_in"
```

Comment:
please use single-quotes instead of double-quotes. Double-quotes are evaluated in bash and can be a security concern.

Reply:
Done :)